lgc/patch/PatchResourceCollect.cpp

```diff
@@ -1712,37 +1721,48 @@ void PatchResourceCollect::matchGenericInOut() {
     assert(inOutUsage.outputMapLocCount == 0);
     bool generatingColorExportShader = m_shaderStage == ShaderStageFragment;
     generatingColorExportShader &= m_pipelineState->isUnlinked() && !m_pipelineState->hasColorExportFormats();
-    for (auto locMapIt = outLocMap.begin(); locMapIt != outLocMap.end();) {
-      auto &locMap = *locMapIt;
+
+    for (auto locInfoMapIt = outLocInfoMap.begin(); locInfoMapIt != outLocInfoMap.end();) {
+      const unsigned origLocation = locInfoMapIt->first.getLocation();
+      auto &newLocationInfo = locInfoMapIt->second;
+
       if (m_shaderStage == ShaderStageFragment) {
-        unsigned location = locMap.first;
         if (!generatingColorExportShader &&
-            m_pipelineState->getColorExportFormat(location).dfmt == BufDataFormatInvalid) {
-          locMapIt = outLocMap.erase(locMapIt);
+            m_pipelineState->getColorExportFormat(origLocation).dfmt == BufDataFormatInvalid) {
+          locInfoMapIt = outLocInfoMap.erase(locInfoMapIt);
           continue;
         }
       }
 
-      if (m_shaderStage == ShaderStageGeometry) {
-        if (locMap.second == InvalidValue) {
-          unsigned outLocInfo = locMap.first;
-          mapGsGenericOutput(*(reinterpret_cast<InOutLocationInfo *>(&outLocInfo)));
```

Comment:
this was the only call to mapGsGenericOutput, so it can be deleted if it's no longer used here

Reply:
Thanks. Done.